jonp2020/desk-booking-app
# Database management

This adds jest tests
It connects to the DB in the jest test file
It also adds ability to use json and urlencoded data
It adds the routes, model and controller for api/users

To test, pull this branch
In the terminal run npm test
4 tests should run, 1 test should pass (the test for api/users returning a status of 200)
